MiniSim/MenuItems/SubMenuItem.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  SubMenuItem.swift
+//  MiniSim
+//
+//  Created by Anton Kolchunov on 11.10.23.
+//
+
+import Cocoa
+import Foundation
+
+protocol SubMenuItem { }
+
+protocol SubMenuActionItem: SubMenuItem {
+    var title: String { get }
+    var tag: Int { get }
+    var needBootedDevice: Bool { get }
+    var bootsDevice: Bool  { get }
+    var image: NSImage? { get }
+}
+
+enum SubMenuItems {
+    enum Tags: Int {
+        case copyName = 100
+        case copyID
+        case coldBoot
+        case noAudio
+        case toggleA11y
+        case paste
+        case delete
+        case customCommand = 200
+    }
+    
+    struct Separator: SubMenuItem { }
+    
+    struct CopyName: SubMenuActionItem {
+        let title: String = NSLocalizedString("Copy name", comment: "")
```

Comment:
Do we need those explicit typing here? Let's remove them from all items

Reply:
Not really, thanks for pointing out!